src/pages/signup/signup.css

```
@@ -0,0 +1,174 @@
+.flexBox {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.mainContainer {
+  width: 100%;
+  max-width: 618px;
+}
+
+.logoContainer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 24px;
+  margin-bottom: 24px;
+}
+
+.logoImg {
+  width: 104px;
+  height: 104px;
+}
+
+.logoTextImg {
+  width: 266px;
+  height: 90px;
+}
+
+.inputContainer {
+  height: 98px;
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+  margin-bottom: 24px;
+}
+
+.inputContainer label {
+  font-family: Pretendard, sans-serif;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 26px;
+  color: var(--secondary-800);
+}
+
+.inputContainer input {
+  width: 640px;
+  height: 56px;
+  line-height: 26px;
+
+  text-indent: 24px;
+
+  border: none;
+  border-radius: 12px;
+
+  background-color: var(--cool-gray-100);
+}
+
+.inputContainer input::placeholder {
+  font-family: Pretendard, sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+}
+
+.passwordWrapper {
+  position: relative;
+  width: 640px;
+  height: 56px;
+}
+
+.passwordWrapper input {
+  width: 100%;
+  height: 100%;
+  line-height: 26px;
+  text-indent: 24px;
+
+  border: none;
+  border-radius: 12px;
+  background-color: var(--cool-gray-100);
+}
+
+.togglePassword {
+  position: absolute;
+  top: 50%;
+  right: 16px;
+  transform: translateY(-50%);
+
+  border: none;
+  background: none;
+  cursor: pointer;
+  padding: 0;
+}
+
+.togglePassword img {
```

❗️ 수정요청
이미지에 크기를 제한해주었지만 img는 자체적으로 vertical-align: baseline 을 기본으로 가지고 있어 의도대로 정렬되지 않습니다. 아래처럼 작성해주세요~

```suggestion
.togglePassword img {
  vertical-align: bottom;
```

[mdn](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/img#styling_with_css)